applications/FluidDynamicsApplication/custom_conditions/navier_stokes_wall_condition.cpp

```diff
@@ -41,8 +41,8 @@ void NavierStokesWallCondition<TDim,TNumNodes,TWallModel...>::EquationIdVector(
 
     IndexType local_index = 0;
     const auto& r_geometry = GetGeometry();
-    const unsigned int v_pos = r_geometry[0].GetDofPosition(DENSITY);
-    const unsigned int p_pos = r_geometry[0].GetDofPosition(MOMENTUM);
+    const unsigned int v_pos = r_geometry[0].GetDofPosition(VELOCITY_X);
+    const unsigned int p_pos = r_geometry[0].GetDofPosition(PRESSURE);
```

Comment:
Shouldn't this belong to another PR?

Reply:
It was already merged in #12335 (I think I should merge the master in here).